.github/workflows/performance_testing.yml

```diff
@@ -0,0 +1,27 @@
+name: Performance testing
+
+on:
+  push:
+    branches: [ main, release* ]
+  pull_request:
+    branches: [ main, release* ]
```

Comment:
Do we want to adjust those? Our release branches are named differently?

Reply:
Yes, thanks for catching that. I'll align it with the targets for the unit tests workflow

Reply:
Done.